### ISSUE TITLE
[Bug] Emoji picker appear blurred for some reason

### ISSUE THREAD

**Issue by yazae** [BDX-town/Mangane#267]
Hi, I’m on the last akkoma version, with the last Mangane version (just installed the two from source).

I have a bit of a problem with the web interface : the emoji picker is kinda blurred while anything else is clear. 
(Sorry for the shitty screen with the emoji bug but it’s not linked to the bug, I tried removing the emoji pack and it is still blurred)
![image](https://github.com/BDX-town/Mangane/assets/88883564/ea9c99fb-e840-413a-8434-51e00c6739e2)

Is it a known bug ?

Yazae

**Comment by yazae** [comment]
I can’t access to the server right now but I’ll add a non bugged screen when my box will be working again.

**Comment by yazae** [comment]
idk why but disabling --tw-rotate or the other items in red make the menu appear and move the emoji picker but resolve the blurr problem, if it helps :
![image](https://github.com/BDX-town/Mangane/assets/88883564/c96eb6ec-2316-4f4a-a42d-e4c9ccb54953)

**Comment by Cl0v1s** [comment]
Hi ! Thanks a lot for your researches about this issue :)

Maybe it's my own screen but I dont exactly see what's blurry in the screen you posted. 
But I admit the details your provided look similar to issues I had at work with complexes UI and transform property

**Comment by yazae** [comment]
Yeah I know it’s really subtle but it’s a thing who put me off when I see it, the interface is so pretty otherwise… I promise it’s here :’D
You can see the difference with the text mainly, but the emojis are subtely blurred as well.

I tried on Edge and the blurr is still here so it’s not a browser issue I think.

--
I found another weird bug, my "s" and "n" key don’t works in the emoji picker research field 🤔 
How can I do a proper bug report ? 
Is this field (hedgedoc-something) the ID of the installed release or it’s something else ?
![image](https://github.com/BDX-town/Mangane/assets/88883564/bc76e4c0-10ad-430e-af8a-5e9552155f77)

**Comment by Cl0v1s** [comment]
Thanks for all those details ! 
Edge is based on chromium, as well as google chrome. What is the browser you first noticed the issue in ? 

About the s and n key, I think there is already an open issue about it, at least I'm aware of it :D

**Comment by yazae** [comment]
My primary browser is Firefox. That’s why I tested it on Edge as well :)

Okay for the s and n keys !

**Comment by Cl0v1s** [comment]
Ok thanks ! So it's a rendering issue, probaly directly related to the code ! 

I'll look on that with your great hints. Thanks again !
